packages/storybook/cxl-ui/cxl-playbook-accordion/theme=cxl-playbook.faq.stories.js

```diff
@@ -0,0 +1,55 @@
+import { html } from 'lit-html';
+import '@conversionxl/cxl-ui/src/components/cxl-app-layout.js';
+import { unsafeHTML } from 'lit-html/directives/unsafe-html';
+import faqData from '../cxl-vaadin-accordion/theme=cxl-faq.data.json';
+import '@vaadin/vaadin-ordered-layout';
+
+export default {
+  title: 'CXL UI/cxl-playbook-accordion',
+};
+
+export const CXLPlaybookAccordionThemeFaq = () => html`
+  <style>
+    .plural .entry-title {
+      margin: 0;
+    }
+    cxl-playbook-accordion {
+      padding: 10px;
+    }
+  </style>
+
+  <cxl-app-layout theme="2c-l" id="container">
```

Comment:
If just displaying the accordion contents, can skip cxl-app-layout container for now.

cxl-app-layout is meant to be used for displaying full app context, and we're not entirely there yet.

Reply:
done 092f192